modules/renderer.js

```diff
@@ -40,18 +42,19 @@ function centerCircle(canvas) {
  * **DEPRECATED FUNCTION** (Although, useful for testing...)
  * @param {SVG} canvas 
  */
+/*
 function drawNumbers(canvas) {
```

Comment:
is this draw numbers in seconds? hours? min?, a bit confused.

Reply:
None. It simply counts from 1 - 9. Ends with 0.  It will transition every second until reaching zero. It's a deprecated function, so it'll get removed eventually.